src/repo.rs

```diff
@@ -269,6 +269,34 @@ impl Repo {
         Ok(self.resolve_rev(refspec, false)?.unwrap())
     }
 
+    /// Query metadata for a content object.
```

Comment:
This docstring may read better with an additional `///` break to split the summary from the body.